includes/Database_Upgrader.php

```diff
@@ -29,6 +29,8 @@
 /**
  * Class Database_Upgrader
  *
+ * @SuppressWarnings(PHPMD.ExcessiveClassComplexity)
```

Comment:
I think this is safe to ignore this error here.

Reply:
Works for me. Although with the growing size here it would make sense to at some point move each migration into its own class.

Similar to what Yoast does: https://github.com/Yoast/wordpress-seo/tree/f997e15c296eea16f55ea242346365b446a8c993/src/config/migrations

Reply:
💯 I was going to post this as a comment myself. Should be simple to put into place, if just make this list a list of class names instead of methods. But I think that require us to move it to its own namespace to keep things tidy.

Reply:
Ticket created: #5668